week1_intro/submit.py

```diff
@@ -24,7 +24,17 @@ def submit_interface(policy, email, token):
 
 
 def submit_taxi(generate_session, policy, email, token):
-    sessions = [generate_session(policy) for _ in range(100)]
+    try:
+        env = gym.make('Taxi-v3')
+    except gym.error.DeprecatedEnv:
+        # Taxi-v2 was replaced with Taxi-v3 in gym 0.15.0
+        env = gym.make('Taxi-v2')
+
+    try:
+        sessions = [generate_session(env, policy) for _ in range(100)]
+    finally:
+        env.close()
+
```

Comment:
Should probably rewrite it with a `make_env()` function so I can use a context manager here, like I do everywhere else.